onnx/defs/data_type_utils.cc

```diff
@@ -112,6 +112,9 @@ const TypeProto& DataTypeUtils::ToTypeProto(const DataType& data_type) {
   std::lock_guard<std::mutex> lock(GetTypeStrLock());
   auto it = GetTypeStrToProtoMap().find(*data_type);
   assert(it != GetTypeStrToProtoMap().end());
```

Comment:
Should we remove the `assert`? It seems redundant now.

Reply:
Done.